aip/general/0131.md

```diff
@@ -71,7 +71,7 @@ message GetBookRequest {
 
 - A resource name field **must** be included. It **should** be called `name`.
```

Comment:
Is there a reason the name of the field is allowed to vary - or is that a **should** as well?

Reply:
Personally, I don't think it should be changed, but regardless let's save that for another PR?